### PR TITLE
Terraform upgrade mp

### PR DESCRIPTION
**Pull request by mark-patton**

### CODE REVIEW

**Comment by mark-patton** [comment]
@m24murray can you give a fresh review to account for fresh updates.  thanks

File: modules/mysql_generalpurpose/main.tf [hunk 1]
```diff
@@ -1,30 +1,31 @@
...
-    tier = "${var.lpg_gp_tier}"
-    family = "${var.lpg_gp_family}"
-  }
+  sku_name = var.lpg_gp_skumname
+ /*  {
+    name = var.lpg_gp_skumname
+    capacity = var.lpg_gp_capacity
+    tier = var.lpg_gp_tier
+    family = var.lpg_gp_family
+  }*/
```
Comment (m24murray) [inline modules/mysql_generalpurpose/main.tf]: Any reason this is commented?

Comment (mark-patton) [reply modules/mysql_generalpurpose/main.tf]: hi Matt - good spot - it should be removed.  In the latest version of tf, the formation of this block has changed.  It was commented-out for testing.  I'll remove and push the change.

Comment (mark-patton) [reply modules/mysql_generalpurpose/main.tf]: @m24murray this has now been updated in this branch.  Please can you review.

**Pull request merged**